packages/Webkul/Category/src/Database/Seeders/CategoryBulkTableSeeder.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Webkul\Category\Database\Seeders;
+
+use Faker\Generator as Faker;
+use Illuminate\Database\Seeder;
+use Webkul\Category\Repositories\CategoryRepository;
+
+/*
+ * CategoryBulkTableSeeder
+ *
+ * Command: php artisan db:seed --class=Webkul\\Category\\Database\\Seeders\\CategoryBulkTableSeeder
+ *
+ * This seeder has not included anywhere just for development purpose.
+ */
+class CategoryBulkTableSeeder extends Seeder
+{
+    private $numberOfParentCategories = 10;
+
+    private $numberOfChildCategories = 50;
+
+    public function __construct(
+        Faker $faker,
+        CategoryRepository $categoryRepository
+    )
+    {
+        $this->faker = $faker;
+        $this->categoryRepository = $categoryRepository;
+    }
+
+    public function run()
+    {
+        for ($i=0; $i<$this->numberOfParentCategories; ++$i) {
```

Comment:
Use space before and after with binary operator

Reply:
Done